src/overlays/actors/ovl_En_Mag/z_en_mag.c

```diff
@@ -533,9 +541,18 @@ void EnMag_DrawInner(Actor* thisx, PlayState* play, Gfx** gfxP) {
         FILENAME_UPPERCASE('L'), FILENAME_UPPERCASE('L'), FILENAME_UPPERCASE('E'), FILENAME_UPPERCASE('R'),
     };
     static u8 pressStartFontIndices[] = {
-        FILENAME_UPPERCASE('P'), FILENAME_UPPERCASE('R'), FILENAME_UPPERCASE('E'), FILENAME_UPPERCASE('S'),
-        FILENAME_UPPERCASE('S'), FILENAME_UPPERCASE('S'), FILENAME_UPPERCASE('T'), FILENAME_UPPERCASE('A'),
-        FILENAME_UPPERCASE('R'), FILENAME_UPPERCASE('T'),
+#if !PLATFORM_IQUE
+        FILENAME_UPPERCASE('P'),
+        FILENAME_UPPERCASE('R'),
+        FILENAME_UPPERCASE('E'),
+        FILENAME_UPPERCASE('S'),
+        FILENAME_UPPERCASE('S'),
+#endif
+        FILENAME_UPPERCASE('S'),
+        FILENAME_UPPERCASE('T'),
+        FILENAME_UPPERCASE('A'),
+        FILENAME_UPPERCASE('R'),
+        FILENAME_UPPERCASE('T'),
```

Comment:
Maybe worth a comment "For iQue, the word "press" is a chinese texture, drawn below"